commitlint.config.js

```
@@ -0,0 +1,34 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
```

ESLint

[error] Replace `'@commitlint/config-conventional'` with `"@commitlint/config-conventional"`

(prettier/prettier)